cvm-attestation/install.sh

```diff
@@ -29,10 +29,20 @@ retry_command sudo apt-get update
 echo "Installing tpm2-tools and Python..."
 retry_command sudo apt-get install -y tpm2-tools python3 python3-pip
 
+# Detect Ubuntu version
+UBUNTU_VERSION=$(lsb_release -sr)
+
+# Set pip install command
+if [[ "$UBUNTU_VERSION" == "24.04" ]]; then
+    PIP_INSTALL_CMD="sudo -H pip3 install --break-system-packages"
+else
+    PIP_INSTALL_CMD="sudo -H pip3 install"
+    retry_command $PIP_INSTALL_CMD --upgrade pip
```

Comment:
should this retry_command be outof the if else block?

Reply:
the issue is that with 24.04 this command breaks too, so I just run it only on non 24.04 version.